algorithms/closest_pair.py

```diff
@@ -84,7 +84,12 @@ def _distance_between(point_1, point_2):
 
```

Comment:
https://github.com/Wynndow/algorithms/pull/2/files#diff-01016d6362c2e04e50a730472494b7c2R35

I think you can just do the min of `closest_pair_from_halves` and `closest_split_pair` for the return value here.